timeline/templates/timeline/_search_results.html

```diff
@@ -0,0 +1,74 @@
+{% load rules %}
+{% load timeline_tags %}
+
+{# Projectroles dependency #}
+{% load projectroles_common_tags %}
+
+<style type="text/css">
+  .table#sodar-tl-search-table thead tr th:nth-child(2) {
+    width: 75px !important;
+  }
+
+  .table#sodar-tl-search-table tbody tr td:nth-child(2) {
+    width: 75px !important;
+  }
+
+  .table#sodar-tl-search-table thead tr th:nth-child(4) {
+    width: 75px !important;
+  }
+
+  .table#sodar-tl-search-table tbody tr td:nth-child(4) {
+    width: 75px !important;
+  }
+
+  /* Responsive modifications */
+  @media screen and (max-width: 1300px) {
+    .table#sodar-tl-search-table tr th:nth-child(4),
+    .table#sodar-tl-search-table tr td:nth-child(4) {
+      display: none;
+    }
+  }
+
+  @media screen and (max-width: 1100px) {
+    .table#sodar-tl-search-table tr th:nth-child(2),
+    .table#sodar-tl-search-table tr td:nth-child(2) {
+      display: none;
+    }
+  }
+
+  @media screen and (max-width: 900px) {
+    .table#sodar-tl-search-table tr th:nth-child(5),
+    .table#sodar-tl-search-table tr td:nth-child(5) {
+      display: none;
+    }
+  }
+
+  @media screen and (max-width: 600px) {
+    .table#sodar-tl-search-table tr th:nth-child(3),
+    .table#sodar-tl-search-table tr td:nth-child(3) {
+      display: none;
+    }
+  }
+</style>
+
+{% if search_results.all.items|length > 0 %}
+  {% include 'projectroles/_search_header.html' with search_title=search_results.all.title result_count=search_results.all.items|length %}
+
+  <table class="table table-striped sodar-card-table sodar-search-table" id="sodar-tl-search-table">
+    <thead>
+      <tr>
+        <th>Timestamp</th>
+        <th>Event</th>
+        <th>Description</th>
+      </tr>
```

Comment:
Looks good, a few things I would like to see in this template:
- Add column for `user`
- Add the project badge for events with a project, similar to the site views. Maybe make the project badge an include template so we don't have repetition?